### PR TITLE
Model Viewer Asset Screenshots

### PR DESCRIPTION
**Pull request by vivianliu0**
Loom Demo:
https://www.loom.com/share/1df692086af24e1aa36eae08e3fa6e03

### CODE REVIEW

File: src/Components/Admin/Assets/assetCard.js [hunk 1]
```diff
@@ -79,7 +79,12 @@ export default function AssetCard({ data, handleDeleteClick }) {
             >
                 <img
                     className={classes.cardImage}
```
Comment (ahmedhamodi) [inline src/Components/Admin/Assets/assetCard.js:81]: Given the different heights/widths of the default image and of the screenshot, let's add an attribute to cardImage.

If you add: `objectFit: contains` or `objectFit: covers` (please test which one looks better locally). You can find more about object-fit property here: https://www.w3schools.com/css/css3_object-fit.asp

Can you also retroactively add this single prop to the other cards (scenarioCard and sceneCard)? Thanks :)

Comment (ahmedhamodi) [reply src/Components/Admin/Assets/assetCard.js:81]: Wanted to add an image to show you what I mean. The armchair is too tall, so using object-fit should make it look better. 
image

**Review by jaydhulia — APPROVED** [review]
This LGTM!

**Comment by ahmedhamodi** [comment]
I assume there's nothing we can do about assets that load side ways right @vivianliu0?

Ie. this sunframe3
image

**Comment by vivianliu0** [comment]
> I assume there's nothing we can do about assets that load side ways right @vivianliu0?
> 
> Ie. this sunframe3
> image

Yeah :/ not sure we can do anything about that

**Review by ahmedhamodi — APPROVED** [review]
lgtm, tested locally :)

**Pull request merged**